omegaconf/_utils.py

```diff
@@ -321,32 +322,29 @@ class ValueKind(Enum):
     VALUE = 0
     MANDATORY_MISSING = 1
     INTERPOLATION = 2
-    STR_INTERPOLATION = 3
 
 
-def get_value_kind(value: Any, return_match_list: bool = False) -> Any:
+def get_value_kind(value: Any, return_parse_tree: bool = False) -> Any:
```

Comment:
Definitely!
Done in 5b4f2f8 and ac8e2e7. The first commit improves `_get_value()` so that we can directly re-use it in `get_value_kind()`. It didn't break anything and I added tests for the new functionalities.